plc4j/api/src/main/java/org/apache/plc4x/java/api/value/PlcBYTE.java

```diff
@@ -169,6 +169,30 @@ public PlcBYTE(@JsonProperty("value") short value) {
         }
     }
 
+    @Override
+    @JsonIgnore
+    public boolean isBoolean() {
+        return true;
+    }
+
+    @Override
+    @JsonIgnore
+    public boolean getBoolean() {
+        return (value != null) && !value.equals(0);
+    }
```

Comment:
In general a bit-string (at least in my perception of it, represents an array of booleans. So if for example I read one byte, from the point of the API it should be a list of 8 boolean values ... I think we need to allow this somehow. Right now the only way would be to get the "byte" value and to pick that appart.

Reply:
I was having trouble with how we would do this. I was initially thinking that if we request to read a WORD we should parse the return value as a PlcList of PlcBOOLs instead of a PlcWORD but wasn't sure how to implement this easily.

But your idea of returning a list from a getBoolean method seems reasonable.